code/game/gamemodes/game_mode.dm

```diff
@@ -516,3 +516,75 @@
 		return 0
 
 	return max(0, enemy_minimum_age - C.player_age)
+
+
+/datum/game_mode/proc/greetAntagonist(var/datum/mind/M, var/antagonist)
+	var/list/validAntagonists = list("traitor", "double agent", "changeling", "nuclear operative", "wizard", "head revolutionary", "blob", "cultist", "shadowling", "gang boss", "abductor scientist", \
+									 "abductor agent", "space ninja", "revenant")
+	if(!(antagonist in validAntagonists))
+		return 0
+	var/text
+	text += "<font color='red' size=3><center><b>You are the [antagonist].</b></center></font><br><br>"
+	switch(antagonist)
+		if("traitor")
+			text += "<center>You are an agent of the Syndicate aboard [station_name()].</center>"
+			text += "<center>You are to accomplish the following objectives assigned to you by your employers:</center><br>"
+			var/objCount = 1
+			for(var/datum/objective/objective in M.objectives)
+				text += "<center><b>Objective #[objCount]:</b> [objective.explanation_text]</center>"
+				objCount++
+			text += "<br><center>You have also been supplied with an uplink cunningly hidden in your PDA or headset. This uplink contains 20 telecrystals and allows you to purchase exclusive equipment \
+			to aid in completing your objectives. In addition, you have been given a set of code words to identify other agents. Use these in conversation but be aware that anyone may be a potential foe. \
+			</center><br><br>"
+			text += "<center><b>Code Phrases:</b> <font color='red'>[syndicate_code_phrase]</font></center>"
+			text += "<center><b>Code Responses:</b> <font color='red'>[syndicate_code_response]</font></center><br><br>"
+			text += "<center>Accomplish your objectives at all costs!</center>"
+			text += "<center><i><b>Wiki Page:</b></i> <a href='https://tgstation13.org/wiki/Traitor'>Click Here</a></center>"

```

Comment:
I'd use the ```config.wikiurl``` variable instead of hardcoding our wiki. So it would be ```<a href='[config.wikiurl]/Traitor'>``` Same goes with the others.